net/http/authz/store.go

```diff
@@ -75,6 +75,48 @@ func (s *Store) Save(ctx context.Context, g *Grant) sinkdb.Op {
 	)
 }
 
+// SaveAll returns an Op to store all the grants
+// Duplicates are ignored
+// It also sets field CreatedAt to the time the grants
+// are stored, or the time the original grant was stored
+func (s *Store) SaveAll(ctx context.Context, grants []*Grant, policy string) sinkdb.Op {
+	if len(grants) == 0 {
+		return sinkdb.Op{}
+	}
+	key := s.keyPrefix + policy
+	var grantList GrantList
```

Comment:
Maybe consider naming this something like `existing`? Then some of the comparisons below would be a little easier to read.